packages/plugin-relay/src/schema-builder.ts

```diff
@@ -409,6 +409,7 @@ schemaBuilderProto.connectionObject = function connectionObject(
       pageInfo: t.field({
         ...pageInfoFieldOptions,
         type: this.pageInfoRef(),
+        nullable: false,
```

Comment:
This should probably be before `pageInfoFieldOptions` so it can be overwritten.

Reply:
Actually, I can just do this before I release, fixing a couple other things anyway.